client/src/main/java/model/Fleet.java

```diff
@@ -1,10 +1,14 @@
 package model;
 
+import gui.fields.Field;
+
 import java.util.ArrayList;
+import java.util.Iterator;
 import java.util.List;
 import java.util.stream.Collectors;
 
-public class Fleet {
+
+public class Fleet implements Iterable<Ship>{
```

Comment:
do you really need implement Iterable?

Reply:
I think is better to implement iterable than creating getter which will return list of ships, but if my way of thinking is correct I dont know :)